content/docs/tls/_index.md

```diff
@@ -48,6 +49,24 @@ The expiration threshold of the Vault TLS certificate in Go Duration format.
 
 Default value: *168h*
 
+## Helm chart custom TLS settings
+
+The following attributes influence the TLS settings of the Helm chart. The `ca.crt` key is mandatory in [secretName](#secretname), otherwise the Bank-Vaults components can't verify the Vault server certificate.
```

Comment:
I'd add something like this to point out that this is a new feature not available in earlier versions:

"Starting with version 1.20, the Vault Helm chart allows you to set custom TLS settings."

Reply:
Added, thanks!